src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -394,21 +385,21 @@ IEnumerable<Config> GetAotConfigs ()
 
 					var libs = new List<string>();
 					if (NdkUtil.UsingClangNDK) {
-						libs.Add ($"-L{GetShortPath (toolchainLibDir)}");
-						libs.Add ($"-L{GetShortPath (androidLibPath)}");
+						libs.Add ($"-L{toolchainLibDir}");
```

Comment:
I'm concerned about the apparent lack of quoting here.  What happens if (when) `toolchainLibDir` contains spaces?  This would presumably become (on Windows) `-LC:\Some Path\With Spaces`.  Will LLVM be able to deal with this?

Doubly worrying is that we effectively do:

    aotOptions.Add ("ld-flags="    + string.Join (";", libs).Replace (@"\", @"\\"));

I have no idea what this would do on Windows, *particularly* when spaces are introduced.

Reply:
I'll add a unit test with spaces and non-ascii characters for the Aot tests to make sure this all works